src/resources/elements/NetworkFeedback/NetworkFeedback.ts

```diff
@@ -1,17 +1,49 @@
 import { autoinject, containerless, customElement } from "aurelia-framework";
-import { EthereumService } from "services/EthereumService";
+import { AllowedNetworks, EthereumService, Networks } from "services/EthereumService";
+import { BrowserStorageService } from "services/BrowserStorageService";
 
 @autoinject
 @containerless
 @customElement("networkfeedback")
 
 export class NetworkFeedback {
 
-  private network: string;
-  private isTestNet;
+  private network: AllowedNetworks;
+  private isProduction;
+  private show: boolean;
 
-  constructor(private ethereumService: EthereumService) {
+  constructor(
+    private ethereumService: EthereumService,
+    private storageService: BrowserStorageService,
+  ) {
     this.network = EthereumService.targetedNetwork;
-    this.isTestNet = EthereumService.isTestNet;
+    this.show = false;
+
+    this.isProduction = process.env.NODE_ENV !== "development" || process.env.NETWORK === "mainnet";
+    const storedNetwork = this.storageService.lsGet<AllowedNetworks>("network");
+    if (
+      storedNetwork &&
+      (this.isProduction && ![Networks.Mainnet, Networks.Celo, Networks.Arbitrum].includes(storedNetwork))
+      || (!this.isProduction && ![Networks.Rinkeby, Networks.Alfajores, Networks.Kovan].includes(storedNetwork))
```

Comment:
was this copy paste from Daoism?
If not, would like to have the some constant to name what the booleans are doing, eg
`const xyz = this.isProduction && ![Networks.Mainnet, Networks.Celo, Networks.Arbitrum].includes(storedNetwork)`